test/e2e/data/shared/v1.6/metadata.yaml

```diff
@@ -1,6 +1,15 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+  - major: 1
+    minor: 7
+    contract: v1beta1
```

Comment:
```suggestion
```

This should not contain v1.7. (the metadata.yaml of the 1.6 release does not contain the 1.7 release)